src/othermbc.md

```diff
@@ -40,70 +41,75 @@ nonsensical.
+consisting of a 74LS377 octal latch in addition to the ROM chip. Because
 the mapper consists of a single standard 74 series logic chip, it has
 two unusual properties:
 
-First, unlike a usual MBC, it switches the whole 32 kiB ROM area instead
-of just the $4000-$7FFF area. If you want to use the interrupt vectors
+First, unlike a usual MBC, it switches the whole 32 KiB ROM area instead
+of just the $4000-$7FFF area. Therefore, if you want to use [the interrupt vectors](<#Interrupt Handling>)
 with this cart, you should duplicate them across all banks.
 Additionally, since the initial state of the '377 can't be guaranteed,
 the ROM header and some code for switching to a known bank should also
 be included in every bank. This also means that the Wisdom Tree mapper
-could be used as a multicart mapper for 32 kiB ROMs, assuming there was
+could be used as a multicart mapper for 32 KiB ROMs, assuming there was
 enough ROM space in each bank for some small initialization code, and
 none of the ROMs wrote to the $0000-$7FFF area. For example, if the
 last 5 bytes of all banks are unused, games can be patched as follows:
 
-    ; At $0100 in all banks but the first
-      nop
-      jp $7FFB
-
-    ; At $7FFB in all banks
-      ld hl, $0100
-      ld [hl], a
-      jp hl
-
-Second, because the '377 latches data on the *positive* edge, and the
-value on the Game Boy data bus is no longer valid when the positive edge
-of the write pulse arrives, the designer of this mapper chose to use the
+```rgbasm
+; At $0100 in all banks but the first
+    nop
+    jp $7FFB
+```
+```rgbasm
+; At $7FFB in all banks
+    ld hl, $0100
+    ld [hl], a
+    jp hl
+```
+
+Second, because the '377 latches data on the *positive* write pulse edge,
+and the value on the Game Boy data bus is no longer valid when the
+positive edge arrives, the designer of this mapper chose to use the
 A7-A0 address lines for selecting a bank instead of the data lines.
 Thus, the value you write is ignored, and the lower 8 bits of the
 address is used. For example, to select bank $XX, you would write any
 value to address $YYXX, where $YY is in the range $00-$7F.
 
-An emulator can detect a ROM designed for Wisdom Tree mapper in one of
+An emulator can detect a ROM designed for the Wisdom Tree mapper in one of
 two ways:
 
--   ROM contains "WISDOM TREE" or "WISDOM\\x00TREE" (the space can
-    be $20 or $00), $0147 = $00, $0148 = $00, size \> 32k. This
-    method works for the games released by Wisdom Tree, Inc.
--   $0147 = $C0, $014A = $D1. These are the values recommended by
-    beware for 3rd party developers to indicate that the ROM is
-    targeting Wisdom Tree mapper hardware. (See below.)
+- [ROM title](<#0134-0143 - Title>) is "WISDOM TREE" (the space may be a
+  $00 NUL character instead), $0147 = $00, $0148 = $00, size \> 32k.
+  This method works for the games released by Wisdom Tree, Inc.
+- $0147 = $C0, $014A = $D1. These are the values recommended by
```

Comment:
"recommended by beware"  is not clear and such phrases should be avoided.
Who is "beware"? Is it even clear we are speaking of a person? 

"A standard proposed by BGB emulator proposes $0147 = $C0, $014A = $D1 to indicate this mapper as a target".

But even phrased this way. What standard? Where is the proposal? And under which assumptions it's a good choice? What's the rationale about the proposal? We should discuss and decide:

1. A place where to put those "proposals"
2. How to reference

Reply:
Well, this should have been addressed when this info was merged; I'm not even touching this part.

Reply:
It wasn't merged. It was part of the baseline from where we started working (the wiki version):
https://gbdev.gg8.se/wiki/articles/Memory_Bank_Controllers#Magic_values_for_detection_of_multicarts_in_emulators

I think we move this issue in another PR as we are out of scope, but beware (lol) it'll be more time consuming to face those issues separatly (as we already pointed them down) and some of them are definitely blocking for this PR, so I'd honestly prefer switch this PR to draft and widen its scope.